scripts/testTimezones.js

```diff
@@ -0,0 +1,118 @@
+﻿/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Prefer just doing this for a line; then actual mistakes will still be caught in this file